src/AssetLoader.php

```diff
@@ -241,6 +255,13 @@ public function withJsDefaultOptions(array $jsDefaultOptions): self
         return $new;
     }
 
+    public function withJsDefaultPosition(?int $position): self
```

Comment:
Probably need to specify in docblock what `int ' stands for, what possible values are.

Reply:
Done.